### DISCUSSION TITLE
Section to provide explanation when dismissing Dependabot alerts.

### DISCUSSION THREAD

**Discussion by RLburrito** [community/community#13363] [Code Security]
Please add section to provide explanation when dismissing Dependabot alerts especially in cases where none of the reasons available apply and have reasoning appear in the Closed alerts section. Adding "Other" would also be okay.

**Comment by mdubard** [comment]
Plus one! An other section where users can provide custom notes on why they are dismissing it, or more insight into why the risk is mitigated elsewhere, would definitely be useful

**Comment by mapsam** [comment]
This would be excellent! When marking alerts as "Risk is tolerable to this project" it's nice to know _why_ that is the case. This is especially important for security teams so they can ensure downstream teams have done their due-diligence investigating the issue and confirming that the vulnerability isn't exploitable.

Appreciate all the work on Dependabot!
